### PR TITLE
fix rust rtfm `base_url`

### PR DESCRIPTION
**Pull request by Tom-the-Bomb**
!!!!!!!!

### CODE REVIEW

**Comment by jay3332** [comment]
💣

File: cogs/rtfm.py [hunk 1]
```diff
@@ -207,7 +207,7 @@ async def rust(self, ctx, *, query: str | None = None) -> str | None:
         """
         Search Rust standard library documentation.
         """
-        base_url = 'https://doc.rust-lang.org/std/'
+        base_url = 'https://doc.rust-lang.org'
```
Comment (Cryptex-github) [inline cogs/rtfm.py:210]: Tom bomb this will break fetch tom bomb see L175

**Review by asze17 — CHANGES_REQUESTED** [review]

**Comment by Cryptex-github** [comment]
Fixed in commit d7c6c25f709167dcec48820bdf390b4dca26a271

**Comment by Tom-the-Bomb** [comment]
thou have not fixed the regex grammar

**Pull request closed without merging**